prime-router/src/main/kotlin/azure/WorkflowEngine.kt

```diff
@@ -308,23 +308,35 @@ class WorkflowEngine(
             )
             val ids = tasks.map { it.reportId }
             val reportFiles = ids
-                .map { db.fetchReportFile(it, org = null, txn = txn) }
+                .mapNotNull {
+                    try {
+                        db.fetchReportFile(it, org = null, txn = txn)
+                    } catch (e: Exception) {
+                        println(e.printStackTrace())
+                        // sanityCheckReport below will log the problem in better detail.
```

Comment:
This comment confused me.  It sounds like there is a sanity check below, but it only says null.

Reply:
done